services/src/main/groovy/org/jfrog/artifactory/client/impl/ArtifactorySystemImpl.groovy

```diff
@@ -34,12 +34,18 @@ class ArtifactorySystemImpl implements ArtifactorySystem {
 
     @Override
     String configuration() {
-        return artifactory.get(getSystemConfirmationApi(), String, null);
+        return artifactory.get(getSystemConfigurationApi(), String, null);
     }
 
     @Override
     void configuration(String xml) {
-        artifactory.post(getSystemConfirmationApi(), ContentType.APPLICATION_XML, xml, null, String, null)
+        artifactory.post(getSystemConfigurationApi(), ContentType.APPLICATION_XML, xml, null, String, null)
+    }
+
+    @Override
+    void patchConfiguration(String yml) {
+        ContentType mimeType = ContentType.create("application/x-yaml");
```

Comment:
This should be `ContentType mimeType = ContentType.create("application/yaml")`. Artifactory 6.1.0 returns an HTTP `415 Unsupported Media Type` when `Content-Type:application/x-yaml` is sent.